app/validators/xml_validator.rb

```diff
@@ -16,8 +16,10 @@ def xsd(record)
 
   def xsd_location(record)
     return "manifest.xsd" if record.is_a?(Resource) || record.is_a?(CustomManifest)
+    return "training.xsd" if record.is_a?(Tip)
     return record.resource.resource_type.dtd_file if record.is_a?(Page) || record.is_a?(TranslatedPage)
     return record.page.resource.resource_type.dtd_file if record.is_a?(CustomPage)
+    return record.tip.resource.resource_type.dtd_file if record.is_a?(CustomTip)
```

Comment:
`CustomTip` should use the same xsd as `Tip`

Reply:
Good catch, thanks